templates/platform_landing_zone/modules/management_groups/locals.tf

```diff
@@ -13,4 +13,12 @@ locals {
       overrides               = try(policy_assignment_value.overrides, null)
     } }
   } }
+  default_retries = {
```

Comment:
You might want to consider adding the ALZ module default values here too as this papers over a few other cracks!